backend/inference.py

```
@@ -318,52 +278,175 @@ def _get_engine():
+    if file_type == "video":
+        records, meta = eng.predict_video(path, frame_rate)
+        agg = aggregate_frames([r["pFake"] for r in records])
+        any_face = any(r["facesFound"] > 0 for r in records)
+        score = float(agg["score"])
+        if not any_face:
+            score = 0.5
+
+        prediction = "deepfake" if score > 0.5 else "real"
+        confidence = (
+            50 if not any_face
+            else int(round(max(score, 1.0 - score) * 100))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)